autotst/conformer/systematic.py

```diff
@@ -168,19 +174,30 @@ def opt_conf(i):
                 except rmgpy.exceptions.AtomTypeError:
                     logging.info("Could not create a RMG Molecule from optimized conformer coordinates...assuming not isomorphic")
                     return False
-        
+        convirged = False
```

Comment:
typo here

Reply:
I think this is fixed with the latest commit